File: task-03/HelloWorld.py
def is_prime(n):

  if n < 2:
    return False
  for i in range(2, int(n**0.5) + 1):
    if n % i == 0:
      return False
  return True
def print_primes(n):
  for i in range(2, n + 1):
    if is_prime(i):
      print(i)
if __name__ == "__main__":
  n = int(input("Enter a number: "))
  print_primes(n)

File: task-03/HelloWorld.cpp
#include <stdio.h>
#include <stdlib.h>
int is_prime(int n) {
  if (n < 2) {
    return 0;
  }
  for (int i = 2; i <= (int)sqrt(n); i++) {
    if (n % i == 0) {
      return 0;
    }
  }
  return 1;
}
void print_primes(int n) {
  for (int i = 2; i <= n; i++) {
    if (is_prime(i)) {
      printf("%d\n", i);
    }
  }
}
int main() {
  int n;
  printf("Enter a number: ");
  scanf("%d", &n);
  print_primes(n);
  return 0;
}